port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -99,7 +119,7 @@ namespace port_drayage_plugin
             ptree pt;
             std::istringstream strategy_params_ss(msg->strategy_params);
             boost::property_tree::json_parser::read_json(strategy_params_ss, pt);
-            std::string mobility_operation_cmv_id = pt.get<std::string>("cmv_id");
+            unsigned long mobility_operation_cmv_id = pt.get<unsigned long>("cmv_id");
```

Comment:
Nitpick, but for whatever it's worth, the sizeof(unsigned long) is implementation/compiler/architecture specific, so this may result in some undefined behavior if we ever attempt to run this on something with a smaller u-long than our 4-byte IDs. Maybe not worth anything more than a comment, but food for thought I suppose.

Reply:
Added a comment to note this in the code.